### PR TITLE
Specify tuspy version explicitly.

### PR DESCRIPTION
**Pull request by Onebrownsound**
What:
* Makes `upload` example Python 3 friendly.
* Changes `tuspy` back to `0.2.1` as an exact match.

Why:
* Internal `tuspy` library changes will not function with Python 2.7 out of the box, without modifications.

Tidbits:

* The [line](https://github.com/tus/tus-py-client/blob/v0.2.3/tusclient/request.py#L60) in question which was blowing up for `tuspy`  versions `>=0.2.2`.
* The [line](https://github.com/tus/tus-py-client/blob/v0.2.1/tusclient/request.py#L70) which serves the same function in `tuspy` version `0.2.1`.

The difference between them, is the `TusRequest.handle` attribute. For `0.2.1` it is a `pycurl` instance, which appears to be able to handle unicode characters in a Python `2.7` string.
While the `TusRequest.handle` attribute for versions `>=0.2.2`  is an `http.client.HTTPSConnection` instance which appears to not function for Python `2.X`, even if at the IO boundary we specify the file encoding as `utf-8`.

The screenshot below, demonstrates an upload (which was failing in the same manor as various user reports) is now succeeding. 

screen shot 2018-09-06 at 4 35 44 pm

TL;DR This should fix the various user reports that have been received.

@erunion

### CODE REVIEW

File: example/upload.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 # Create a variable with a hard coded path to your file system
 file_name = '<full path to a video on the filesystem>'
 
-print 'Uploading: %s' % file_name
+print('Uploading: %s' % file_name)
```
[the remark this answers is not in the document]
Comment (Onebrownsound) [reply example/upload.py]: This example wouldn't run in Python 3, because print took on the signature of a function`print 'lkjds` -> `print('blah')`. This change will function for both 2 && 3.
